src/annotate/celltypist/config.vsh.yaml

```diff
@@ -38,7 +38,11 @@ argument_groups:
         min: 1
         description: | 
           The minimum number of genes present in both the reference and query datasets.
-  
+      - name: "--sanitize_gene_names"
```

Comment:
Does this only remove version numbers? In this case I would use

`--var_names_remove_ensembl_version_number` or something like that